rott/rt_draw.h

```diff
@@ -59,8 +59,8 @@ extern int fandc;
 // math tables
 //
 extern short    tantable[FINEANGLES];
-extern fixed    sintable[FINEANGLES+FINEANGLEQUAD+1];
-extern fixed    *costable;
+extern int    sintable[FINEANGLES+FINEANGLEQUAD+1];
```

Comment:
Strictly, these tables should be of `fixed` type.

Reply:
When I set it to fixed type, I get an error in the BuildTables function, which seems like it might have some logical errors:
```
~/taradino/rott/rt_draw.c: In function ‘BuildTables’:
~/taradino/rott/rt_draw.c:240:23: error: passing argument 1 of ‘SwapIntelLongArray’ from incompatible pointer type [-Wincompatible-pointer-types]
  240 |    SwapIntelLongArray(&sintable[0], length);
      |                       ^~~~~~~~~~~~
      |                       |
      |                       fixed * {aka long int *}
In file included from ~/taradino/rott/rt_draw.c:40:
~/taradino/rott/rt_util.h:73:32: note: expected ‘int *’ but argument is of type ‘fixed *’ {aka ‘long int *’}
   73 | void  SwapIntelLongArray (int *l, int num);
      |                           ~~~~~^
```

This code over there seems a bit off, since it does not use the fixed type, but int:
```
//
// get size of sin/cos table
//

   memcpy(&length,ptr,sizeof(int));
   SwapIntelLong(&length);
   ptr+=sizeof(int);
   
//
// get sin/cos table
//
   memcpy(&sintable[0],ptr,length*sizeof(int));
   SwapIntelLongArray(&sintable[0], length);
   ptr+=(length)*sizeof(int);
```

Reply:
Right, it gets more and more fishy.